subcommands/targets/add.go

```diff
@@ -169,11 +169,11 @@ func deriveTargets(factory string, hwIds map[string]interface{}, srcTag string,
 	// We assume that the initial OSTree Target(s) is/are created by the OTA service during Factory creation,
 	// hence we don't need to create a new Target from scratch in this command, just derive Targets from existing one.
 	if len(latestTargetsPerHwId) == 0 {
-		return nil, fmt.Errorf("no any source Targets to derive new Targets from are found; source tag: %s", srcTag)
+		return nil, fmt.Errorf("no source Targets to derive new found; source tag: %s", srcTag)
 	}
 	for id := range hwIds {
 		if _, ok := latestTargetsPerHwId[id]; !ok {
-			return nil, fmt.Errorf("no source Target to derive new Target from is found"+
+			return nil, fmt.Errorf("no source Target to derive new found"+
```

Comment:
I'm not sure the wording of these two sentences makes sense. @vkhoroz might have a suggestion

Reply:
Yeah. Maybe "No source found to derive new Target from" or "Could not find source Target to derive new Target from"?

Reply:
@doanac I updated it, hopefully it is more understandable? This is assuming that my understanding of what was originally being said is correct.

Reply:
@kprosise It looks fine to me now.